components/heatmapcontrol/heatmapcontrol.js

```diff
@@ -277,6 +285,11 @@ HeatmapControl.prototype.__opacityChange = function() {
   }
 };
 
+// createSelect         - creates options for each field
+// createField          - creates fields and their sliders
+// createOpacities      - creates opacity field and it's slider
+// createIntervalInputs - creates HTML Color inputs for given noOfIntervals
```

Comment:
Can you put these right before their appropriate functions?

Reply:
Done